app/javascript/mastodon/actions/suggestions.js

```diff
@@ -48,5 +48,7 @@ export const dismissSuggestion = accountId => (dispatch, getState) => {
     id: accountId,
   });
 
-  api(getState).delete(`/api/v1/suggestions/${accountId}`);
+  api(getState).delete(`/api/v1/suggestions/${accountId}`).then(() => {
+    fetchSuggestions()(dispatch, getState);
```

Comment:
Should this not be called through dispatch?

Reply:
I wrote the contents of fetchSuggestions inline.